specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/examples/CloudEndpoints_Create.json

```diff
@@ -25,27 +25,28 @@
             "body": {
                 "name": "SampleCloudEndpoint_1",
                 "properties": {
+                    "backupEnabled" : "false",
```

Comment:
use boolean false here instead of string

Reply:
but this is indeed a boolean in swagger https://github.com/Azure/azure-rest-api-specs/blob/2cf4198df15d2fb24f6643065f79677b05706915/specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2018-10-01/storagesync.json#L3274